### PR TITLE
Declutter the "Getting started" chapter (step 1)

### PR DESCRIPTION
**Pull request by DelazJ**
This is a first step to make the "Getting started" chapter simpler and welcoming. The idea is to bring it to only show how to download, run (the common way) and follow simple steps to produce a minimal map.

This PR:
- moves the "[Advanced command lines](https://docs.qgis.org/testing/en/docs/user_manual/introduction/getting_started.html#advanced-command-line-options)" section to QGIS Configuration page. I considered moving to QGIS Gui but not very fan. If anyone has a better suggestion...
- completes  the sample session with map output

In the next steps, "[generating output](https://docs.qgis.org/testing/en/docs/user_manual/introduction/getting_started.html#generating-output)" and "[introducing QGIS projects](https://docs.qgis.org/testing/en/docs/user_manual/introduction/getting_started.html#introducing-qgis-projects)" should also move elsewhere --- again if anyone has suggestions...

### CODE REVIEW

File: source/docs/user_manual/introduction/getting_started.rst [hunk 1]
```diff
@@ -1,586 +1,350 @@
...
+#. Click on the |symbology| :guilabel:`Symbology` tab and select blue as fill color.
+#. Click on the |labeling| :guilabel:`Labels` tab and select :guilabel:`Single labels`
+   in the drop-down menu to enable labeling. Then from the :guilabel:`Label with`
+   list, choose the ``NAMES`` field to label each lake with its name.
+#. To improve readability of labels, you can add a white buffer around them
+   by clicking :guilabel:`Buffer` in the list on the left, checking |checkbox|
+   :guilabel:`Draw text buffer` and choosing ``3`` as buffer size.
+#. Click :guilabel:`Apply`. Check if the result looks good, and finally click
+   :guilabel:`OK`.
+#. Select :menuselection:`View --> Decorations --> Scale Bar` menu and in the
```
Comment (havatv) [inline source/docs/user_manual/introduction/getting_started.rst]:
```suggestion
#. Select :menuselection:`View --> Decorations --> Scale Bar`, and in the
```

File: source/docs/user_manual/introduction/getting_started.rst [hunk 2]
```diff
@@ -1,586 +1,350 @@
...
+   dialog that opens, check |checkbox| :guilabel:`Enable Scale Bar` option and
```
Comment (havatv) [inline source/docs/user_manual/introduction/getting_started.rst]:
```suggestion
   dialog that opens, check |checkbox| :guilabel:`Enable Scale Bar` and
```

File: source/docs/user_manual/introduction/getting_started.rst [hunk 3]
```diff
@@ -1,586 +1,350 @@
...
+   press :guilabel:`Apply` (you can customize some options of the dialog if
+   you want).
+#. Likewise, from the decorations menu, add more items (north arrow, copyright...)
+   to the map canvas
+#. Click :menuselection:`Project --> Import/Export -->` |saveMapAsImage|
+   :menuselection:`Export Map to Image...` and press :guilabel:`Save` in the
+   opened dialog. Select a file location, a format and confirm by pressing
+   :guilabel:`Save` again.
+#. Press :menuselection:`Project -->` |fileSave| :menuselection:`Save...` to
```
Comment (havatv) [inline source/docs/user_manual/introduction/getting_started.rst]:
```suggestion
#. Use :menuselection:`Project -->` |fileSave| :menuselection:`Save...` to
```

**Comment by DelazJ** [comment]
No new text in the forced push; due to some LF/CRLF configuration conflicts between the different OSes I use, the whole file was considered as changed instead of some lines. Fixed now.

**Comment by havatv** [comment]
Looks good.

**Pull request merged**